components/01-visual-styling/05-branding-standards/01-global-header/global-header.scss

```diff
@@ -12,15 +12,15 @@ a.Skip-btn:focus {
 }
 #skip-content {
 	position: absolute;
-	z-index: 0;
+	z-index: -1;
```

Comment:
let's have this match /president and make it overflow: hidden and left: -100000px